src/App.js

```diff
@@ -1,6 +1,12 @@
 import React from 'react';
-import './App.css';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/Home';
+import Quote from './components/Quote';
 
 class App extends React.Component {
   constructor(props) {
```

Comment:
- [OPTIONAL] This should be a functional component and not a class component, Kindly change this I should looks like this 👇🏻 

```javascript
function App() {
  return (
    <div className="app">
      <Router>
        <Routes>
          <Route exact path="/">
            <Home />
          </Route>
          <Route exact path="/album/:calculator">
            <Calculator />
          </Route>
          <Route exact path="/quote">
            <Quotes />
          </Route>
        </Routes>
      </Router>
    </div>
  );
}

export default App;
```

Reply:
Thanks @BenMukebo , I shall fix this.